packages/core/ios/RNEmbrace/SpanRepository.swift

```diff
@@ -50,12 +50,12 @@ class SpanRepository {
     func spanStarted(span: Span) -> String {
         let key = getKey(span)
 
-        if activeSpans.count > MAX_STORED_SPANS {
-            os_log("too many active spans being tracked, ignoring", log: log, type: .error)
-            return ""
-        }
-
         activeSpansQueue.async(flags: .barrier) {
+            if self.activeSpans.count > MAX_STORED_SPANS {
```

Comment:
there's a count check below for `if completedSpans.count > MAX_STORED_SPANS {` that this strategy should also get applied to